### DISCUSSION TITLE
Add support for wingman/shortmatch/longmatch

### DISCUSSION THREAD

**Discussion by DigitalFriends** [PhlexPlexico/G5V#109] [Ideas]
Hi!

As of now, i am running multiple serverdirectories for different type of servers, would it be possible to add support for different gamemodes in match creation or season creation? Such as wingman, shortmatch, longmatch. 

As of now there is a "workaround", but it requires a server to be specifically "hardcoded" for the mode that is needed, because i need to make changes to /cfg/get5/live.cfg.

**Comment by PhlexPlexico** [comment]
With the way get5 handles max rounds and win conditions I don't think this one is possible. You could try doing overrides in the match cvar fields, but I'm almost positive that won't work... I'm not exactly sure how else you would/could override values from the get5 configs that are on the game server :(

**Reply by DigitalFriends** [reply]
Right, as of now i'm running two server installations pararell with custom start parameters, cfg/get5/live_wingman.cfg and custom sourcemod config folders. 

The thing is, when deploying many servers like this, diskspace starts becoming an issue :D 

But thank you for looking into it!
